### PR TITLE
[ENG-642] Run samples end-to-end tests on autokitteh PRs

### PR DESCRIPTION
**Pull request by tebeka**
See ENG-642

### CODE REVIEW

File: .github/workflows/ci-samples.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+name: Samples End to End Test
```
Comment (daabr) [inline .github/workflows/ci-samples.yml]:
```suggestion
name: Samples End-to-End Test
```

File: .github/workflows/ci-samples.yml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+      - main
+  pull_request:
+
+jobs:
+  build-and-test:
+    name: Samples e2e
+    runs-on: ubuntu-latest
+    timeout-minutes: 10
+    steps:
+      - name: Check out code
```
Comment (daabr) [inline .github/workflows/ci-samples.yml]:
```suggestion
      - name: Check out samples repo
```

File: .github/workflows/ci-samples.yml [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+        uses: actions/checkout@v4
+        with:
+          repository: https://github.com/autokitteh/samples.git
+      - name: Set up Go
+        uses: actions/setup-go@v5
+        with:
+          go-version-file: tests/go.mod
+      - name: Get autokitteh
+        run: curl -sSL https://get.autokitteh.sh | bash
```
Comment (daabr) [inline .github/workflows/ci-samples.yml]:
```suggestion
        run: curl -fsSL https://get.autokitteh.sh | bash
```

Comment (tebeka) [reply .github/workflows/ci-samples.yml]: Fixed

**Comment by tebeka** [comment]
Already merged into #216

**Pull request closed without merging**